Database/Updates/World/01-06-19-2017-alterPostionTable.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE `ace_world`.`ace_position`   
+  DROP COLUMN `landblock` ,
+  ADD COLUMN `landblock` INT(5) UNSIGNED AS ( ((landblockraw >> 16) & 0xFFFF)) VIRTUAL,
```

Comment:
Can also change back to smallint with this code
``landblock` smallint(5) GENERATED ALWAYS AS ((`landblockRaw` >> 16)) VIRTUAL,`
as the previous expression was forcing the change to an INT for whatever reason